Xamarin.Forms.Platform.WPF/Converters/ViewToRendererConverter.cs

```diff
@@ -1,37 +1,20 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Globalization;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Xamarin.Forms.Platform.WPF
 {
 	public sealed class ViewToRendererConverter : System.Windows.Data.IValueConverter
 	{
 		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
 		{
-			VisualElement visualElement = value as VisualElement;
-			if (visualElement != null)
+			var visualElement = value as VisualElement;
+			if (visualElement == null)
 			{
-				var frameworkElement = Platform.GetOrCreateRenderer(visualElement)?.GetNativeElement();
-
-				if(frameworkElement != null)
-				{
-					frameworkElement.Loaded += (sender, args) =>
-					{
-						visualElement.Layout(new Rectangle(0, 0, frameworkElement.ActualWidth, frameworkElement.ActualHeight));
```

Comment:
this code is executed in CellControl.cs

Reply:
But can't this be used in other controls? or was just on cells?

Reply:
I have found this converter only works with Cell (see xaml file)
```
	<DataTemplate x:Key="ViewCell">
		<ContentPresenter Content="{Binding View, Converter={StaticResource ViewToRenderer}}" />
	</DataTemplate>
```